src/python/engagement-creator/src/engagement-creator.py

```diff
@@ -330,3 +353,8 @@ def lambda_handler(events: Any, context: Any) -> None:
             LOGGER.error(f"mainloop exception {e}")
             LOGGER.error(traceback.print_exc())
             time.sleep(2)
+
+
+if __name__ == "__main__":
+    if IS_LOCAL:
```

Comment:
weird check - does this just not run in prod or what

Reply:
Agree 100% this is weird, I think we can fix it by running lambdas in https://hub.docker.com/r/lambci/lambda/ for local grapl. I plan to test it out in the work I'm doing on analyzer deployments.

Reply:
cool. in the meantime, just keeping this documented :)